cmd/bee-join/main.go

```diff
@@ -153,6 +161,7 @@ Will output retrieved data to stdout.`,
 	c.Flags().StringVar(&host, "host", "127.0.0.1", "api host")
 	c.Flags().IntVar(&port, "port", 8080, "api port")
 	c.Flags().BoolVar(&ssl, "ssl", false, "use ssl")
+	c.Flags().IntVarP(&loglevel, "loglevel", "l", 0, "log verbosity")
```

Comment:
in `bee` this is called `verbosity`, let's keep the same convention across the codebase